test/pytest/test_browser.py

```diff
@@ -1,10 +1,12 @@
+import contextlib
 import glob
 import os
 import subprocess
-import sys
-from typing import Iterable
+from typing import AsyncIterator
 
 import pytest
+from webdriver_bidi import ChromiumBidi, WebdriverBidi
+from yarl import URL
```

Comment:
We happen to have python3-yarl-1.9.2-5.fc40.x86_64 in our tasks container already, as dependency of aiohttp.